region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -366,6 +366,39 @@
       "requires": [],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 7],
+      "name": "Spikesuit - R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"canShineCharge": {"usedTiles": 20, "openEnd": 1}},
+        "canShinechargeMovementTricky",
+        "canBeVeryPatient",
+        {"thornHits": 1},
+        "canTrickyCarryFlashSuit",
+        "canSpikeSuit",
+        "Morph",
+        {"enemyKill": {
+          "enemies": [["Beetom"]],
+          "explicitWeapons": ["ScrewAttack", "Missile", "PowerBomb"]
```

Comment:
This list of weapons was used in the farming strats; it leaves out Supers though I can't remember if there is a good reason why. In any case, if we have an `enemyKill` option here it should probably just use `"excludedWeapons": ["Bombs"]`. But there would also be a `canTrickyDodgeEnemies` option that avoids the Beetoms and gets them stuck out of the way; at this level it could maybe just be expected, but should at least be an option.